contrib/masterfiles-stage/README.org

```diff
@@ -86,7 +90,7 @@ The following parameters are in use, be sure to avoid their collision
 when developing new staging methods.
 
 - VCS_TYPE - The staging method to use
-  - Currently supported: (GIT|GIT_TAG_OR_COMMIT|SVN)
+  - Currently supported: (GIT|GIT_TAG_OR_COMMIT|GIT_POLICY_CHANNELS|SVN)
```

Comment:
I'd add a note here that `GIT_TAG_OR_COMMIT` is legacy and equivalent to the other one.